app/routes/user/endorsements.server.ts

```diff
@@ -8,11 +8,12 @@
 
 import { tables } from '@architect/functions'
 import {
+  AdminAddUserToGroupCommand,
```

Comment:
Remember to add this to the arc.codes plugin that sets the Lambda's IAM actions.